modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1363,6 +1363,152 @@ definition {
+				fieldBusinessType = "Text",
+				fieldLabelName = "Custom Field",
+				fieldName = "customObjectField",
+				fieldType = "String",
+				isRequired = "true",
+				objectName = "CustomObject185");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject185");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 185");
+
+			ObjectAdmin.goToActionsTab();
+
+			ObjectAdmin.addObjectActionViaUI(
+				actionLabel = "Action Label",
+				actionName = "ActionName",
+				active = "true",
+				errorMessage = "Error",
+				inputMethod = "true",
+				inputValue = "New Entry Test",
+				objectLabel = "Custom Object 185",
+				thenAction = "Add an Object Entry",
+				whenAction = "Standalone");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customObjectField",
+				objectName = "CustomObject185",
+				value = "Entry Test");
+		}
+
+		task ("And: A new user is added") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+		}
+
+		JSONRole.addRegularRole(roleTitle = "Regular Role");
+
+		Role.openRolesAdmin();
+
+		LexiconEntry.gotoEntry(rowEntry = "Regular Role");
+
+		Navigator.gotoNavItem(navItem = "Define Permissions");
+
+		ObjectAdmin.searchInDefinePermissions(search = "Custom Objects 185");
+
+		Click(
+			key_wikiPageVersionNew = "action.ActionName",
+			locator1 = "WikiTable#SELECT_VERSION_NEW_VERSION_CHECKBOX");
+
+		Click(
+			key_wikiPageVersionNew = "Access in Control Panel",
+			locator1 = "WikiTable#SELECT_VERSION_NEW_VERSION_CHECKBOX");
+
+		Click(
+			key_wikiPageVersionNew = "View",
+			locator1 = "WikiTable#SELECT_VERSION_NEW_VERSION_CHECKBOX");
+
+		Click(
+			key_wikiPageVersionNew = "Permissions",
+			locator1 = "WikiTable#SELECT_VERSION_NEW_VERSION_CHECKBOX");
+
+		PortletEntry.save();
+
+		var objectId = JSONObject.getObjectId(objectName = "CustomObject185");
+
+		Permissions.definePermissionViaJSONAPI(
+			resourceAction = "VIEW_CONTROL_PANEL",
+			resourceName = 90,
+			roleTitle = "Regular Role",
+			roleType = "regular");
+
+		Permissions.definePermissionViaJSONAPI(
+			resourceAction = "ACCESS_IN_CONTROL_PANEL",
+			resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectId}",
+			roleTitle = "Regular Role",
+			roleType = "regular");
+
+		Permissions.definePermissionViaJSONAPI(
+			resourceAction = "VIEW",
+			resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectId}",
+			roleTitle = "Regular Role",
+			roleType = "regular");
+
+		Permissions.definePermissionViaJSONAPI(
+			resourceAction = "VIEW",
+			resourceName = "com.liferay.object.model.ObjectDefinition#${objectId}",
+			roleTitle = "Regular Role",
+			roleType = "regular");
+
+		task ("When the user go to set permissions in object definition permissions") {
```

Comment:
When the user is given permission to modify the permissions of a standalone action